index.html

```diff
@@ -49,7 +49,7 @@
 
    
           <label for="tip">Tip Percentage:</label>
-          <input type="range" min="0" max="100" value="0" step="1" onchange="tipCalculate(this.value, 'bill')" class="range-style">
+          <input id="slider" type="range" min="0" max="100" value="0" step="1" onchange="updateTip(this.value, 'bill')" class="range-style">
```

Comment:
Here I add an "id" to the slider which I use to locate it later on in the script.js file. I also change it to call a copy of the original tipCalculate function.